slack/web/async_client.py

```diff
@@ -2156,3 +2174,154 @@ async def views_publish(
         else:
             kwargs.update({"view": view})
         return await self.api_call("views.publish", json=kwargs)
+
+    async def admin_conversations_create(
+        self,
+        is_private: bool,
+        name: str,
+        *,
+        org_wide: bool = False,
+        team_id: str = None,
+        **kwargs
+    ) -> AsyncSlackResponse:
+        """Create a public or private channel-based conversation.
+
+        Args:
+            is_private (bool): When true, creates a private channel instead of a public channel
+            name (str): Name of the public or private channel to create.
+            org_wide (bool): When true, the channel will be available org-wide.
+                Note: if the channel is not org_wide=true, you must specify a team_id for this channel
+            team_id (str): The workspace to create the channel in.
+                Note: this argument is required unless you set org_wide=true.
+
+        """
+        if not org_wide and team_id is None:
+            raise e.SlackRequestError("team_id is required if org_wide is False")
+        kwargs.update(
+            {
+                "is_private": is_private,
+                "name": name,
+                "org_wide": org_wide,
+                "team_id": team_id,
+            }
+        )
+        return await self.api_call("admin.conversations.create", json=kwargs)
+
+    async def admin_conversations_delete(
+        self, channel_id: str, **kwargs
+    ) -> AsyncSlackResponse:
+        """Delete a public or private channel.
+
+        Args:
+            channel_id (str): The channel to delete.
+
+        """
+        kwargs.update({"channel_id": channel_id})
+        return await self.api_call("admin.conversations.delete", json=kwargs)
+
+    async def admin_conversations_invite(
+        self, channel_id: str, user_ids: Union[str, List[str]], **kwargs
```

Comment:
Have `*` consistently

Reply:
I wanted to double check before I changed them all but should it be 
```python
def api_method(self, *, ....) ->: 
```
for all the methods or just the two tagged in your review?

Reply:
I meant for all the methods (just hesitated to repeat it for all).